tox.ini

```diff
@@ -11,6 +11,7 @@ deps =
     portend
     pytest-services
     munch
+    mypy
```

Comment:
@jaraco I wondered about using `jaraco.envs` to reuse `[testenv:types]`, instead of adding mypy to `[testenv]deps`. Any preferences/suggestions?

Reply:
Note that this is only relevant if we keep the tests.

Reply:
> Any preferences/suggestions?

I don't really have an opinion on it, other than it _feels_ different than what `jaraco.envs` was written for. If it works, however, I'm happy to support it.